CITATION.cff

```diff
@@ -0,0 +1,63 @@
+cff-version: 1.2.0
+message: "If you'd like to cite Kedro, please use the following metadata"
+authors:
+- family-names: "Bălan"
+  given-names: "Lorena"
+- family-names: "Kunii"
+  given-names: "Kiyo"
+- family-names: "Hoang"
+  given-names: "Lim"
+- family-names: "Ivaniuk"
+  given-names: "Andrii"
+- family-names: "Deriabin"
+  given-names: "Dmitrii"
+- family-names: "Dada"
+  given-names: "Yetunde"
+- family-names: "Datta"
+  given-names: "Deepyaman"
+- family-names: "Patel"
+  given-names: "Zain"
+- family-names: "Danov"
+  given-names: "Ivan"
+- family-names: "Wrigley"
+  given-names: "Gordon"
+- family-names: "Stichbury"
+  given-names: "Jo"
+- family-names: "Khan"
+  given-names: "Nasef"
+- family-names: "Theisen"
+  given-names: "Merel"
+- family-names: "Tsaousis"
+  given-names: "Nikolaos"
+- family-names: "Walker"
+  given-names: "Waylon"
+- family-names: "Milne"
+  given-names: "Antony"
+- family-names: "Nguyen"
+  given-names: "Tam"
+- family-names: "Westenra"
+  given-names: "Richard"
+- family-names: "Carvalho"
+  given-names: "Lais"
+- family-names: "Takeru"
+  given-names: "Sasaki"
+- family-names: "Trevisani"
+  given-names: "Marcelo Duarte"
+- family-names: "Bertoli"
+  given-names: "Sebastian"
+- family-names: "Mawjee"
+  given-names: "Shahil"
+- family-names: "Nijholt"
+  given-names: "Bas"
+- family-names: "Vukolov"
+  given-names: "Dmitry"
+- family-names: "Klein"
+  given-names: "Jiri"
+- family-names: "Fischer"
+  given-names: "Kody"
+- family-names: "Koops"
+  given-names: "Marnix"
```

Comment:
How did you generate the list of names? Some of these names I don't recognise at all, presumably because they were contributors at some point. Probably best to include only the main team?

Reply:
I got them from the Zenodo reference that I think @yetudada populated. Also think the main team is probably better!